docker-compose.yml

```diff
@@ -12,28 +11,30 @@ services:
       POSTGRES_PASSWORD: ${DB_PASSWORD}
     ports:
       - "5432:5432"
-    command: ["postgres", "-c", "log_statement=all"]
+    command: [ "postgres", "-c", "log_statement=all" ]
 
   performance_manager:
     container_name: performance_manager
     env_file: .env
     build:
-      context: performance_manager
+      context: ./python_src
       dockerfile: Dockerfile
     depends_on:
       - local_rds
-    working_dir: /performance_manager
+    working_dir: /lamp
     volumes:
       - ~/.aws:/root/.aws:ro # map credentials to be used by boto3, read-only
+    command: [ "poetry", "run", "performance_manager" ]
```

Comment:
docker compose now uses a single docker image from our package, and launches different pipelines using the command argument. on the terraform side, we also want to add a task command that will do the same thing.

Reply:
So this PR will be blocked by a devops PR to incorporate a `command` call in terraform